### PR TITLE
Complex Contraction CK Bilinear Example

### PR DESCRIPTION
**Pull request by ozturkosu**
This is example of complex tensor contraction based on 4 GEMM.  All imaginary and real tensor are explicitly defined.

### CODE REVIEW

File: example/64_complex_contraction_bilinear/README.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+
+## Run
+```bash
+#arg1: verification (0=no, 1=yes)
+#arg2: initialization (0=no init, 1=integer value, 2=decimal value)
+#arg3: time kernel (0=no, 1=yes)
+./bin/example_contraction_bilinear_xdl_fp32 1 1 1
+```
+
+Result (MI210 )
```
Comment (geyyer) [inline example/64_complex_contraction_bilinear/README.md]: Could you replace it with gfx90a for example?

File: example/64_complex_contraction_bilinear/CMakeLists.txt [hunk 2]
```diff
@@ -0,0 +1,3 @@
+add_example_executable(example_complex_contraction_bilinear_xdl_fp32 complex_contraction_bilinear_xdl_fp32.cpp)
```
Comment (geyyer) [inline example/64_complex_contraction_bilinear/CMakeLists.txt]: Looks like example 64 is already used, could you assign a different number?

**Review by geyyer — COMMENTED** [review]
@ozturkosu, thanks, PR looks good, could you address comments below and it should be good to go!

**Comment by bartekxk** [comment]
Hi @ozturkosu why this example is called as complex? I see that you use real data types, not complex.

File: example/64_complex_contraction_bilinear/README.md (hunk 1 above)
Comment (ozturkosu) [reply example/64_complex_contraction_bilinear/README.md]: sure @geyyer I ll update it asap

**Comment by ozturkosu** [comment]
Hi @bartekxk ,  When it was implemented, CK did not support struct data type therefore image and real part are stored as separate tensor.

File: example/64_complex_contraction_bilinear/CMakeLists.txt (hunk 2 above)
Comment (ozturkosu) [reply example/64_complex_contraction_bilinear/CMakeLists.txt]: I made it example 66

File: example/64_complex_contraction_bilinear/README.md (hunk 1 above)
Comment (ozturkosu) [reply example/64_complex_contraction_bilinear/README.md]: I removed performance information

**Review by illsilin — APPROVED** [review]

**Pull request merged**